pkg/controller/csi_handler.go

```diff
@@ -354,6 +357,22 @@ func (h *csiHandler) hasVAFinalizer(va *storage.VolumeAttachment) bool {
 	return false
 }
 
+// Checks if the PV (or) the inline-volume corresponding to the VA could have migrated from
+// in-tree to CSI.
+func (h *csiHandler) isMigratable(va *storage.VolumeAttachment) bool {
+	if va.Spec.Source.PersistentVolumeName != nil {
+		pv, err := h.pvLister.Get(*va.Spec.Source.PersistentVolumeName)
+		if err != nil {
+			return false
```

Comment:
should we return error instead of false?

Reply:
Yes, makesense.